src/components/ConfirmationBox/ConfirmationBox.js

```diff
@@ -1,14 +1,19 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import labels from '../../utils/labels';
 
 const ConfirmationBox = props => {
 	return (
 		<div>
-			<div>{props.text}</div>
+			<div>{`Is this a ${labels[props.item]}?`}</div>
```

Comment:
can you test this line too actually...?
just a contians text is fine

Reply:
Ive added a text includes 'is this a' so it doesnt fire due to the text on the buttons